### ISSUE TITLE
Can't keep tray icon out of the hidden tray area

### ISSUE THREAD

**Issue by NottMee** [sandboxie-plus/Sandboxie#2593]
### Describe what you noticed and did

1. drag Sandman tray icon out of hidden tray area
2. exit Sandman and restart it from context menu
3. the icon lays in the hidden tray area again
screen gif:
 ![GIF](https://user-images.githubusercontent.com/12641553/211309819-2c117700-5b65-46a7-932c-92fd8022ea05.gif)

### How often did you encounter it so far?

100% when you do it

### Affected program

Not relevant

### Download link

Not relevant

### Where is the program located?

Not relevant to my request.

### Expected behavior

the icon should remeber the status and lays out of the hidden tray area (like other programs did)

### What is your Windows edition and version?

Win11 Pro 22H2 x64

### In which Windows account you have this problem?

User account with secure desktop turned off for UAC prompts.

### Please mention any installed security software

Built-In Windows Defender

### What version of Sandboxie are you running?

Sandboxie Plus 1.6.4 x64

### Is it a new installation of Sandboxie?

I have been using the same version for some time.

### In which sandbox type you have this problem?

Not relevant to my request.

### Can you reproduce this problem on a new empty sandbox?

Not relevant to my request.

### Sandboxie.ini configuration

```shell
#
# Sandboxie configuration file
#

[GlobalSettings]
FileRootPath=D:\Data\Sandbox\%SANDBOX%
KeyRootPath=\REGISTRY\USER\Sandbox_%USER%_%SANDBOX%
IpcRootPath=\Sandbox\%USER%\%SANDBOX%\Session_%SESSION%
Template=WindowsRasMan
Template=WindowsLive
Template=OfficeLicensing
Template=OfficeClickToRun
Template=Listary
Template=Edge_Fix
SeparateUserFolders=n
DefaultBox=DefaultBox

[UserSettings_040E00CF]
SbieCtrl_AutoStartAgent=SandMan.exe
SbieCtrl_EnableAutoStart=y

[DefaultBox]
Enabled=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00ffff,ttl,6
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=9
BoxNameTitle=n
CopyLimitKb=81920
ClosedFilePath=<BlockNetDevices>,InternetAccessDevices
```

**Comment by isaak654** [comment]
Isn't it the same behavior of #1590?

**Comment by NottMee** [comment]
> Isn't it the same behavior of #1590?

I think they are the same, and as in #1590 said, when i run SandMan.exe alone, it can keep itself out of hidden tray area, but every time i run a sandboxed file, it will go back to hidden area

**Comment by isaak654** [comment]
@okrc
I don't think there is an easy way to test your change, because this issue is directly related to the behavior of `Run Sandboxed` context menu entry which I cannot enable on the portable build you've attached. If I try to do so, the checkbox to enable `Run Sandboxed` on `Shell Integration`  is automatically unchecked.

**Comment by isaak654** [comment]
@okrc
I don't think there is an easy way to test your change, because this issue is directly related to the behavior of `Run Sandboxed` context menu entry which I cannot enable on the portable build you've attached. If I try to do so, the checkbox to enable `Run Sandboxed` in `Shell Integration`  is automatically unchecked.

**Comment by isaak654** [comment]
@okrc 
I can confirm that the issue has been resolved for me after installing v1.6.5 as a base, renaming the existing SandMan.* files and applying the SandMan.* files from your custom build. Feel free to open a new pull request here.
